### DISCUSSION TITLE
Instructions for updating

### DISCUSSION THREAD

**Discussion by tasermonkey** [Drumstix42/drumstix42.github.io#6] [General]
I am trying to just get on the same page as what you have.  

I see the instructions to download the Ue4Export tool.  Thats fine, but then you reference 'the export.bat' file.  I don't see one of those in the zip file.  However, the [README of the Ue4Export](https://github.com/CrystalFerrai/Ue4Export#batch-script) mentions:
```
@Ue4Export "C:\Games\MyGame\Content\Paks" "assetlist.txt" "C:\Output"
```

Do  you mean create an `export.bat` file? that is pointing to: `%IcarusDirectory%\Icarus\Content\Paks` as the first argument.  Then to an assetlist.txt as the second, and then where you want to output as the third?

Then that leads to the second question, in the README, it mentions, "Adjust `assetlist.txt` so that `[Texture]` section only includes `Icarus/Content/Assets/2DArt/*`"  ... Then with a commenjt about commenting out the other lines... What `assetlist.txt` are you referring to.  [Ue4Export README](https://github.com/CrystalFerrai/Ue4Export#asset-list) mentions creating this file, that does have a `[Texture]` section, but you make me wonder by the comment if you had a specific one in mind.

Then you mention `assetlist_data.txt`.  Is there a specific file you are referencing here?

**Comment by Drumstix42** [comment]
@tasermonkey Yeah you're right. I mistakenly thought these 3 files came with the zip file release. I either got one pre-packaged with the same zip name, or I created them and just didn't recall that I had done it (likely from the Icarus Modding discord instructions.

I'll update the README to reflect.

export.bat
**Please note the filepath needs to be updated to your game installation folder**
```
@echo off

Ue4Export.exe "C:\Games\Steam\steamapps\common\Icarus\Icarus\Content\Paks" assetlist.txt export
Ue4Export.exe "C:\Games\Steam\steamapps\common\Icarus\Icarus\Content\Data" assetlist_data.txt export/data

for /D %%G in (export\data\*) do move %%G export
rmdir export\data
```

assetlist.txt
```
[Text]
#

[Raw]
#

[Texture]
Icarus/Content/Assets/2DArt/*

```

assetlist_data.txt
```
[Text]
#*
```

Obviously the last file will yield no output, but I was including it because the one I originally found had files/folders included, and I wanted to include the section names for thoroughness.
